theano/gpuarray/dnn_fwd.c

```diff
@@ -172,7 +176,18 @@ APPLY_SPECIFIC(conv_fwd)(PyGpuArrayObject *input, PyGpuArrayObject *kerns,
     } else {
       algo = prev_algo;
     }
-
+    
+    params->conv_algo = algo;
+    // CUDNN7: need to set math type
+    err = cudnnSetConvolutionMathType(desc, params->conv_tensor_op);
```

Comment:
This need a "#ifdef CUDNN>=7" or similar I think. In this file, it is not like in libgpuarray, we include cudnn.h and I think this function don't exist in cudnn v6